lib/rucio/transfertool/fts3.py

```diff
@@ -576,8 +584,16 @@ def initialize(self, session, logger=logging.log):
             elif file_state == FTS_STATE.FAILED and job_state == FTS_STATE.FAILED or \
                     file_state == FTS_STATE.FAILED and not self._multi_sources:  # for multi-source transfers we must wait for the job to be in a final state
                 if self._is_recoverable_fts_overwrite_error(self.request(session), reason, self._file_metadata):
+                    file_meta = self._file_metadata
+                    self.logger(logging.DEBUG,
+                                f'Recovered existing file: {request_id} to {file_meta["dst_rse"]} '
+                                f'({self.external_host} {self._transfer_id})')
                     new_state = RequestState.DONE
                 else:
+                    file_meta = self._file_metadata
+                    self.logger(logging.INFO,
```

Comment:
This will be printed for every single FAILED transfer in atlas and other communities. And we already have 20million logs for poller and receiver per day :/  Even in CMS, I think you are not interested in this log for most transfers.

I would "assume" that you want to print this log message before `record_counter` here : https://github.com/rucio/rucio/pull/5818/files#diff-10fdb713ecac24728050341f83e17dbe7cc3f1f5a317823860677917579caea2R472